src/foam/core/Action.js

```diff
@@ -167,7 +168,19 @@ If empty than no permissions are required.`,
     },
 
     function createIsEnabled$(x, data) {
-      return this.createSlotFor_(x, data, this.isEnabled, this.enabledPermissions);
+      var running = foam.core.SimpleSlot.create({ value: false });
```

Comment:
I think we should check if the runningMap already has a value for data.

If we create two isEnabled$ slots for the same combination of data and Action, then I think we can share running state.  If an action is rendered twice on a single view for instance, then clicking on the action in one button will successfully disable both buttons while running.